src/ws/login.js

```diff
@@ -550,7 +550,7 @@
         if (prompt_data.default)
             ei.value = prompt_data.default;
         ei.setAttribute('type', type);
-        ei.focus();
```

Comment:
please do not just comment it, remove it.

Reply:
:O You're right! I thought I deleted it. My bad! I'll go over what you said.